Ryujinx.HLE/HOS/Services/Ssl/SslService/ISslContext.cs

```diff
@@ -1,7 +1,25 @@
+using Ryujinx.Common.Logging;
+using System;
+
 namespace Ryujinx.HLE.HOS.Services.Ssl.SslService
 {
     class ISslContext : IpcService
     {
-        public ISslContext() { }
+        public ISslContext(ServiceCtx context ) { }
+        
+        [Command(4)]
+        public ResultCode ImportServerPki(ServiceCtx context)
+        {
+            var CertificateFormat = context.RequestData.ReadInt32();
+            var unknown = context.RequestData.ReadBytes(5);
+            UInt64 response = 1;
```

Comment:
The response is the id tied to the certificate your register on your session.
Also use ulong instead of UInt64 here
```suggestion
            ulong certificateId = 1;
```